contracts/GovHub.template

```diff
@@ -152,6 +152,7 @@ contract GovHub is System, IParamSubscriber {
     );
     require(targets.length != 0, "must provide actions");
     require(targets.length <= proposalMaxOperations, "too many actions");
+    require(_allPlatformContracts(targets), "not platform contracts");
```

Comment:
If we assume that proposals only contain platform-contract targets better validate it

Reply:
so I'll revert